pkgs/data/themes/mcos-xfce-edition/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, gtk-engine-murrine }:
+
+stdenv.mkDerivation rec {
+  name = "mcos-xfce-edition";
+
+  src = fetchFromGitHub {
+    owner = "paullinuxthemer";
+    repo = "McOS-XFCE-Edition";
+    rev = "8d90e108377e89359e69d83c83099195b6c98f10";
+    sha256 = "0vvag2gakcsd60ypgkz86s7alhddwk2axgq85v7n4hrda21nhswy";
+  };
+
+#  hash is 0vvag2gakcsd60ypgkz86s7alhddwk2axgq85v7n4hrda21nhswy
```

Comment:
Why to keep this commented lines?

Reply:
Which lines? Comme?
I will delete them.

Reply:
Yes.

Reply:
Done!